File: ass53/Program2.java
/*
2. Write Java program which accept N numbers from user and display all such elements which are divisible by 5. 
Input :  N :   6 
  Elements : 85 66 3 80 93 88  
Output : 85 80 
*/
import java.lang.*;
import java.util.*;
class ArrayDemo
{
    void Even(int arr[])
    {
        int iSum1 = 0;
        for(int i=0; i<arr.length; i++)
        {
            if(arr[i]%5==0)
            {
               
               System.out.println("Divisible 5 is : " +arr[i]);
            }
            
        }
        
    }
}
class Program2 
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Element");
        int size = sobj.nextInt();
        int arr[] = new int[size];
        System.out.println("Enter the Array elements");
        for(int i=0; i<arr.length; i++)
        {
            arr[i] = sobj.nextInt();
        } 
        ArrayDemo dobj = new ArrayDemo();
        dobj.Even(arr);
    }
}

File: ass37/program2.c
/*
2. Write a program which accept string from user and return length of largest word. 
Input :   “Marvellous Multi OS Infosystems” 
Output :   11 
*/

#include<stdio.h>
int CountWord(char *str)
{
    int iMax = 0;
    if(str==NULL)
    {
        return -1;
    }
    
    while(*str!='\0')
    {
    
     if(*str =='')
     {
         while((*str =='')&&(*str!='\0'))
         {
             str++;
         }
     }
     else
     {
         if(str > iMax)
         {
             iMax = str;
         }
     }
    }
    
        printf("%d\n",iMax);
}
int main()
{
 //   int bRet ;
    char arr[50]="Marvellous Multi OS";
    CountWord(arr);
    
    return 0;
}